src/worker/vm/imports.ts

```diff
@@ -2,14 +2,76 @@ import { BigQuery } from '@google-cloud/bigquery'
 import * as contrib from '@posthog/plugin-contrib'
 import * as AWS from 'aws-sdk'
 import crypto from 'crypto'
+import dns from 'dns'
 import * as genericPool from 'generic-pool'
-import fetch from 'node-fetch'
-import * as pg from 'pg'
+import nodeFetch from 'node-fetch'
+import nodePostgres from 'pg'
+import { parse } from 'pg-connection-string'
 import snowflake from 'snowflake-sdk'
+import url from 'url'
 import * as zlib from 'zlib'
 
 import { writeToFile } from './extensions/test-utils'
 
+// 93.184.216.34 = example.com
+const RESTRICTED_IPS = ['93.184.216.34']
```

Comment:
would now be a matter of configuring this

Reply:
This will be hard to maintain - we'll be adding services without this blacklist.

Instead we could blacklist the whole block of IPs in our VPC which is `172.31.0.0/16`